docs/manuals/source/Configuration/Director.rst

```diff
@@ -81,8 +81,18 @@ Only a single type (Backup, Restore, ...) can be specified for any job. If you w
 
 Note, you define only a single Job to do the Full, Differential, and Incremental backups since the different backup levels are tied together by a unique Job name. Normally, you will have only one Job per Client, but if a client has a really huge number of files (more than several million), you might want to split it into to Jobs each with a different FileSet covering only part of the total files.
```

Comment:
```suggestion
Note, you define only a single Job to do the Full, Differential, and Incremental backups since the different backup levels are tied together by a unique Job name. Normally, you will have only one Job per Client, but if a client has a really huge number of files (more than several million), you might want to split it into several Jobs each with a different FileSet covering only part of the total files.
```